fixtures/cli-preprocessor/node_modules/with-args/index.ts

```diff
@@ -0,0 +1,4 @@
+export default function (options) {
```

Comment:
I put this into a package, because it turns out just adding a `with-args.ts` file was causing a non-zero exit code that took forever to debug, due to Knip finding an unused file :^)

Reply:
Oh sorry about that, noted :)